sample-apps/go-sample-app/Dockerfile

```
@@ -0,0 +1,12 @@
+FROM golang:1.17-alpine AS build
+RUN apk update && apk add ca-certificates
+WORKDIR /app
+COPY go.mod .
+COPY go.sum .
+RUN GO111MODULE=on go mod download
```

```suggestion
RUN GO111MODULE=on go mod download
COPY . .
RUN <build command>
```

Need to actually build the executable.